ddl/ddl_api.go

```diff
@@ -1996,6 +1991,17 @@ func resolveAlterTableSpec(ctx sessionctx.Context, specs []*ast.AlterTableSpec)
 	return validSpecs, nil
 }
 
+func isSameTypeMultiSpecs(specs []*ast.AlterTableSpec) bool {
+	specType := specs[0].Tp
+	var count int = 0
+	for _, spec := range specs {
+		if spec.Tp == specType {
```

Comment:
Yes, I want to support the statement like `alter table t add column b int after a, add column c int after b;`, because it's necessary. And there is already test case in `ddl_test.go` `TestAlterTableAddColumns`.